docker-compose.env

```diff
@@ -11,6 +11,7 @@ DOCS_PORT=8001
 # Client
 STORYBOOK_PORT=7001
 CLIENT_PORT=7000
+NODE_IMAGE=node:18.17.1
```

Comment:
I don't think this is necessary. I could kinda see an argument for it if we used it in multiple places but I would expect to find the version of the container I am running in `docker-compose.yml`. If it is all the same could we keep it in that file?

Reply:
Got it, let's remove it then!